third_party/2/ipaddress.pyi

```diff
@@ -0,0 +1,149 @@
+from typing import (Any, Container, Generic, Iterable, Iterator, Optional,
```

Comment:
Could you add this file to check_consistent.py to make sure it stays in sync with third_party/3/ipaddress.pyi?

Reply:
Done.